modules/core/04-channel/keeper/events.go

```diff
@@ -127,11 +127,12 @@ func EmitChannelCloseConfirmEvent(ctx sdk.Context, portID string, channelID stri
 
 // EmitSendPacketEvent emits an event with packet data along with other packet information for relayer
 // to pick up and relay to other chain
+
```

Comment:
Unnecessary line break

```suggestion
// EmitSendPacketEvent emits an event with packet data along with other packet information for relayer
// to pick up and relay to other chain
```